src/transmission.js

```diff
@@ -11,7 +11,7 @@
 import superagent from 'superagent';
 import urljoin from 'urljoin';
 
-const userAgent = "libhoney-js/LIBHONEY_JS_VERSION";
+const USER_AGENT = "libhoney-js/LIBHONEY_JS_VERSION";
```

Comment:
Just spent a *much* longer time than you'd think staring at this file and trying to figure out where we were subbing in the actual version number 😂  ([cairn](https://github.com/honeycombio/libhoney-js/blob/b7d2ecd0099ddd74bd3060b1e81fe50d24023256/gulpfile.js#L10))

Reply:
yeahhhhh, I thought about actually just importing `package.json` and using that.